build/cmake/AddMCRL2Binary.cmake

```diff
@@ -243,6 +266,7 @@ function(_add_mcrl2_binary TARGET_NAME TARGET_TYPE)
       _add_resource_files(${TARGET_NAME} "${ARG_MENUNAME}" "${ARG_DESCRIPTION}" "${ARG_ICON}" SOURCES)
     endif()
     
+    
```

Comment:
Normally it's best to avoid arbitrary changes inside pull requests, but we are not very strict on this anyway so it's fine.